src/install/kubernetes/helm/install-helm.mdx

```diff
@@ -0,0 +1,101 @@
+headingText: Install the Kubernetes integration
+---
+
+New Relic has a bunch of Helm charts for the different components, each with its own set of features for the platform. You can install these components separately, but we really recommend using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. New Relic provides this chart, which acts as a wrapper or a meta-package for the individual charts.
+
+To install `nri-bundle` with Helm follow these steps:
+
+1. Make sure `kubectl` is installed and configured correctly to manage the cluster.
+
+2. Use this command to add the New Relic Helm charts repo:
+    ```shell
+    helm repo add newrelic https://helm-charts.newrelic.com
+    ```
+3. Create a file named []`values-newrelic.yaml`](https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/values.yaml), which will be used to define your configuration:
+
+    ```yaml
+    global:
+      licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+      cluster: _K8S_CLUSTER_NAME_
+    
+    newrelic-prometheus-agent:
+      # Automatically scrape prometheus metrics for annotated services in the cluster
+      # Collecting prometheus metrics for large clusters might impact data usage significantly
+      enabled: true
+    nri-metadata-injection:
+      # Deploy our webhook to link APM and Kubernetes entities
+      enabled: true
+    nri-kube-events:
+      # Report Kubernetes events
+      enabled: true
+    newrelic-logging:
+      # Report logs for containers running in the cluster
+      enabled: true
+    kube-state-metrics:
+      # Deploy kube-state-metrics in the cluster.
+      # Set this to true unless it is already deployed.
+      enabled: true
+    ```
+4. Make sure everything is configured properly in the chart by running the following command. Notice that we're specifying `--dry-run` and `--debug`, so nothing will be installed in this step:
+
+    ```shell
+    helm upgrade --install newrelic-bundle newrelic/nri-bundle \
+    --namespace newrelic --create-namespace \
+    -f values-newrelic.yaml \
+    --dry-run \
+    --debug
+    ```
+    
+    Please notice and adjust the following flags:
+    
+        * `global.licenseKey=YOUR_NEW_RELIC_LICENSE_KEY`: Must be set to a valid <InlinePopover type="licenseKey" /> for your account.
+        * `global.cluster=K8S_CLUSTER_NAME`: It's used to identify the cluster in the New Relic UI, so should be a descriptive value not used by any other Kubernetes cluster configured in your New Relic account.
+        * `kube-state-metrics.enabled=true`: Setting this to `true` will automatically install Kube State Metrics (KSM) for you, which is required for our integration to run. You can set this to false if KSM is already present in your cluster, even if it's on a different namespace.
+        * `newrelic-prometheus-agent.enabled=true`: Will deploy our Prometheus Agent, which automatically collects data from Prometheus endpoints present in the cluster.
```

Comment:
Can you say "Run this to deploy" or "Set this to deploy?" Or some sort of action verb?